### ISSUE TITLE
Pretrained weights

### ISSUE THREAD

**Issue by havaeimo** [cbfinn/maml#24]
Hi,
Would it be possible for  you to share the meta-learners trained weights for 5 way imagenet?
Also, do you have the download link for the mini-imagenet files?
Thanks

**Comment by Hexiang-Hu** [comment]
@cbfinn: Hi Chelsea, I also have the some problem re-producing the exact performance on mini-ImageNet (I got ~45% top-1 acc on the test set for 5 way ImageNet, which has a gap from your reported number of 48.7%). It might due to the difference in hyper-parameter settings. (I am using the default settings of this repo)

Therefore I would like to ask if it would be possible to share the trained model or release the configuration for pre-producing mini-ImageNet experiment. It would be great if either of them is available. Thank you so much in advance.

**Comment by cbfinn** [comment]
For the miniimagenet files, contact Sachin Ravi.
I just pushed a trained model for 5-way, 1-shot MiniImagenet -- see the latest commit.
